### DISCUSSION TITLE
Clickable menu for MS app Removal

### DISCUSSION THREAD

**Discussion by darius-vr** [ChrisTitusTech/winutil#199] [Ideas]
Hi there, I am wondering if it would be possible to add the MS store app removeable setting as a clickable setting so that you can choose which apps from the store you would like to uninstall

**Comment by cgriffiths86** [comment]
I'd love this too as I generally have to use a different app/powershell to remove specific apps. I'd like the granularity of picking each app as I don't necessarily want to wipe out all the MS apps at once.
